### PR TITLE
Ch wh smp 68 general info

### PR DESCRIPTION
**Pull request by chandlerHall47**
pull request for the general info page

### CODE REVIEW

**Review by melissacela — APPROVED** [review]
lgtm!

**Review by cluhrmanSWO — APPROVED** [review]
keep up the good work

File: src/Components/Button/Button.test.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ test('Sign-in button should be visible to user', () => {
     render(<Button/>);
 
     // Get the button element
-    const button = screen.getByTestId('sign-in-button');
-
+    const button = screen.getByTestId('button');
+//changed button test id to 'button'
```
Comment (cluhrmanSWO) [inline src/Components/Button/Button.test.js:10]: This kind of comment should be apart of a commit, not within the code

File: src/Pages/GeneralInfo.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import Button from '../Components/Button/Button';
+import NavHeader from '../Components/NavHeader/NavHeader';
+
+
+function GeneralInfo(){
+    return (
+        <div>
+            <NavHeader isLogoutEnabled={true}/>
+           <Container fluid data-testid="general-info" className="mb-5 mt-3">
+            <Row>
```
Comment (cluhrmanSWO) [inline src/Pages/GeneralInfo.js:12]: be sure to keep proper indentation for easier reading of the hierarchy

File: src/Pages/GeneralInfo.test.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
+import { render, screen, fireEvent} from "@testing-library/react";
+import GeneralInfo from '../Pages/GeneralInfo';
+
+
+
+//test to see if general information page is rendered
```
Comment (cluhrmanSWO) [inline src/Pages/GeneralInfo.test.js:6]: Comments like this aren't necessary as the test() method already provides a description of the test

**Pull request merged**